server/rpc.go

```diff
@@ -309,32 +316,45 @@ func verifyOrchestratorReq(orch Orchestrator, addr ethcommon.Address, sig []byte
 	return orch.CheckCapacity("")
 }
 
+type authBroadcasterResponse struct{}
+
 // authenticateBroadcaster returns an error if authentication fails
-func authenticateBroadcaster(id string) error {
+func authenticateBroadcaster(id string) (*authBroadcasterResponse, error) {
 	if AuthWebhookURL == "" {
-		return nil
+		return nil, nil
 	}
 
 	values := map[string]string{"id": id}
 	jsonValues, err := json.Marshal(values)
 	if err != nil {
-		return err
+		return nil, err
 	}
 	res, err := http.Post(AuthWebhookURL, "application/json", bytes.NewBuffer(jsonValues))
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	body, err := ioutil.ReadAll(res.Body)
 	defer res.Body.Close()
 	if err != nil {
-		return err
+		return nil, err
 	}
 
 	if res.StatusCode < 200 || res.StatusCode >= 300 {
-		return errors.New(string(body))
+		return nil, errors.New(string(body))
+	}
+
+	webhookRes := &authBroadcasterResponse{}
+	if err := json.Unmarshal(body, webhookRes); err != nil {
+		return nil, err
 	}
-	return nil
+
+	return webhookRes, nil
+}
+
+func handleWebhookResponse(res *authBroadcasterResponse, oInfo *net.OrchestratorInfo) *net.OrchestratorInfo {
```

Comment:
Would `handleAuthBroadcasterResponse()` be more descriptive to distinguish this webhook response from webhook responses found elsewhere in the codebase (i.e. on the broadcaster side)?